library/Director/Core/RestApiClient.php

```diff
@@ -260,13 +260,11 @@ protected function processReadBuffer($offset, $pos)
 
     public function disconnect()
     {
-        if ($this->curl !== null) {
-            if (is_resource($this->curl)) {
-                @curl_close($this->curl);
```

Comment:
Is removing the `@` really necessary?

Reply:
No, but it looks superfluous to me. This function cannot throw an error.

Reply:
It cannot? Why? If you're unsure, please restore it.

Reply:
It's not documented, but yes, will restore it.